uwblueprint/community-fridge-kw
# build: Deploy backend

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
Followed CCBC backend deployment (https://github.com/uwblueprint/ccbc/pull/12)
* deployed backend server and postgres db on Heroku (free tier pricing for now)
* managed firebase and nodemailer secrets through Heroku buildpack and config vars respectively
* added yarn build and start scripts to package.json to automate Heroku deployment
* made changes to sequelize options to allow server to connect with deployed db
* updated cors origin list to allow client side deploys and deploy previews to make requests

PLEASE UPDATE YOUR .env file with the Nodemailer credentials. 



## Steps to test
1. Go to https://community-fridge-kw.web.app/ and make sure everything is working fine
2. Sign in with the seeded test user, email: `test@test.org` and pw: `password123`
3. Maybe even try registering a new user (this would add it to the deployed db) and play around with it to make sure it works the same way as it does locally



## What should reviewers focus on?
* 


## Checklist
